### DISCUSSION TITLE
AttributeError: module 'deepmd' has no attribute 'DeepPot'

### DISCUSSION THREAD

**Discussion by warriors1523** [deepmodeling/deepmd-kit#1590] [Q&A]
DeePMD-kit v2.1.0
![image](https://user-images.githubusercontent.com/92622311/159737640-87632d7e-5c12-4909-97d4-765b8d177d86.png)

**Comment by wanghan-iapcm** [comment]
The python interface is changed in v2, please check https://github.com/deepmodeling/deepmd-kit/blob/master/doc/inference/python.md

**Comment by warriors1523** [comment]
Thank you very much. Now it works normally.
----- 原始邮件 -----
发件人：Han Wang ***@***.***>
收件人：deepmodeling/deepmd-kit ***@***.***>
抄送人：Stephen ***@***.***>,  Author ***@***.***>
主题：Re: [deepmodeling/deepmd-kit] AttributeError: module 'deepmd' has no attribute 'DeepPot' (Discussion #1590)
日期：2022年03月24日 13点21分


The python interface is changed in v2, please check https://github.com/deepmodeling/deepmd-kit/blob/master/doc/inference/python.md

&mdash;
